cinema/views.py

```diff
@@ -43,11 +88,38 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def list(self, request, *args, **kwargs):
+        response = super().list(request, *args, **kwargs)
+        return Response(response.data["results"])
```

Comment:
The `list` method overrides the default behavior and removes pagination metadata by returning only `response.data['results']`. Consider returning the full `response.data` to include pagination metadata.